### DISCUSSION TITLE
"a repository that has many stars" does not work in some scenarios

### DISCUSSION THREAD

**Discussion by BeauBouchard** [community/community#20559] [Other Feature Feedback, Questions, & Ideas]
where you change the name, or migrated it to an organization is one of them i noticed. 

If I created a repo, transferred it to an ORG and then that repo becomes popular, there is little credit from this achievement, just wanted to confirm if this is true for anyone else?

**Comment by ikramulkayes** [comment]
I think that GitHub encourages collaboration, which explains why ORG repos receive more visitors than personal public repos. Or, since ORG has members, when an ORG member's profile is visited, the visitor may also check the ORG repos (we've all done it), increasing the repo's exposure and star rating.
